submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionRequestDocDataRepository.kt

```diff
@@ -68,7 +67,7 @@ class SubmissionRequestDocDataRepository(
 
     private fun criteriaArray(filter: SubmissionFilter): Array<Criteria> =
         ImmutableList.Builder<Criteria>().apply {
-            add(where("status").`is`(REQUESTED))
+            add(where("status").ne(PROCESSED))
```

Comment:
I triick I have use in the past to avoid the same bug to happen again is to include static variable in enum so we do not forget to updated when adding new value. 

I suggest to do the same here. Create PROCESSING status set in enum and use here `in` query operator.